cfme/containers/node.py

```diff
@@ -190,5 +190,13 @@ def step(self):
         """Navigate to the Timelines page"""
         self.prerequisite_view.toolbar.monitoring.item_select('Timelines')
 
-# TODO Need Ad hoc Metrics
+
+@navigator.register(Node, 'AdHoc')
+class AdHocMain(CFMENavigateStep):
```

Comment:
**Question**: What do you think about calling this ```class AdHocMetrics``` ? Same for here: https://github.com/ManageIQ/integration_tests/blob/master/cfme/containers/provider/__init__.py#L473